packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -171,7 +171,7 @@ const AuthDefault = ({
               icon={icon}
               className="btn-primary"
               buttonSize={ButtonSize.Large}
-              onClick={() => onSocialClick(provider)}
+              onClick={() => onSocialClick(provider.toLowerCase())}
```

Comment:
Can we maybe lowercase providers in `getKratosProviders` once so we don't have to do it all over the place?

Reply:
They are in lowercase from the requests, the issue is, on some other parts, the label being displayed is used (for example, Google on the button).

Reply:
I agree best to have them lowercase as values and upper them in usecases where needed maybe.
Or have both as value and label?

Reply:
Yeah, makes sense to have both properties so we will never be wrong on the actual value.

Reply:
I've also refactored the property `provider` to `label` for easier understanding.